src/api/l_math_vectors.c

```diff
@@ -296,6 +296,25 @@ static int l_lovrVec2Lerp(lua_State* L) {
   return 1;
 }
 
+static int l_lovrVec2Angle(lua_State* L) {
+  float* v = luax_checkvector(L, 1, V_VEC2, NULL);
+  float* u;
+  float uvec[2];
+  float dot, length_v, length_u;
+  if (lua_type(L, 2) == LUA_TNUMBER) {
+    uvec[0] = lua_tonumber(L, 2);
+    uvec[1] = luax_checkfloat(L, 3);
+    u = uvec;
+  } else {
+    u = luax_checkvector(L, 2, V_VEC2, NULL);
```

Comment:
Very minor: changing the NULL here (and in the Vec4 method) to "vec2 or number" would change the error message to say "vec2 or number expected" instead of "vec2 expected".  I'm not even consistent about doing it everywhere though.

Reply:
I've now corrected all such cases in this source file.